frontend/src/libs/components/radio/radio.tsx

```diff
@@ -0,0 +1,44 @@
+import { useFormController } from '#libs/hooks/hooks.js';
+import {
+  type FormControl,
+  type FormFieldPath,
+  type FormFieldValues,
+} from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+type Properties<T extends FormFieldValues> = {
+  control: FormControl<T, null>;
+  label: string;
+  name: FormFieldPath<T>;
+  onChange: () => void;
+  isChecked?: boolean;
```

Comment:
the same about isChecked - should be taken from field.value
```suggestion
```

Reply:
done